samples/bluetooth/ipsp/README.rst

```diff
@@ -19,22 +19,164 @@ Testing with a Linux host
 =========================
 
 To test IPSP please take a look at samples/net/README, in addition to running
-echo-client you must enable 6LowPAN module in Linux with the
-following commands:
+echo-client you must enable 6LowPAN module in Linux with the following commands
+(as root):
 
 .. code-block:: console
 
-   $ modprobe bluetooth_6lowpan
-   $ echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
+   # modprobe bluetooth_6lowpan
+   # echo 1 > /sys/kernel/debug/bluetooth/6lowpan_enable
```

Comment:
Remember we'll need to change this once the new Linux kernel API is available. @Vudentz knows more about this